src/main/java/xbot/common/controls/sensors/XLaserCAN.java

```diff
@@ -26,8 +29,15 @@ public XLaserCAN(DevicePolice police, DeviceInfo info, String owningSystemPrefix
         inputs = new LaserCANInputsAutoLogged();
     }
 
-    public Distance getDistance() {
-        return inputs.distance;
+    public Optional<Distance> getDistance() {
```

Comment:
⭐ I'm wondering if we still want to expose a way to find out what the previous measurement was even if it's not valid?

Reply:
I thought about that, but it seems too easy to shoot yourself in the foot with such an option.